### DISCUSSION TITLE
How can I find up-trending stocks with small rounding base consolidation?

### DISCUSSION THREAD

**Discussion by dhanghan93** [pranjal-joshi/Screeni-py#135] [Q&A]
When I say up-trend : 50 EMA > 200 EMA.

Small rounding base consolidation:  candles creating a U shape base.

U shape base should have a lower volume compared to the recent previous breakout (weak hands are booking their profits).

Reference image: https://www.tradingview.com/x/GvdQR80k

**Comment by pranjal-joshi** [comment]
It is possible to find U/V shape rounding bottoms with explicit input of maximum number of days (candles) during the screening.
Two highs should be observed during these time period along with other condition checks like 50MA > 200MA and lowest closing is in between 2 highs

I will update here if I implement this in subsequent version updates!
